### PR TITLE
KDS-1530-filter-multi-select-in-filter-bar-is-missing-icons

### PR DESCRIPTION
**Pull request by gyfchong**
## Why

Missing check icon for FilterMS in KAIO

## What

Add check icon

### CODE REVIEW

File: packages/components/src/SVG/icons/Check.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { SVG, SVGProps } from "~components/SVG"
+
+export const Check = (props: Omit<SVGProps, "children">): JSX.Element => (
```
Comment (HeartSquared) [inline packages/components/src/SVG/icons/Check.tsx:4]: Where's your `displayName`? :(

**Review by HeartSquared — APPROVED** [review]

File: packages/components/src/SVG/icons/Check.tsx (hunk 1 above)
Comment (gyfchong) [reply packages/components/src/SVG/icons/Check.tsx:4]: Oh, none of the icons have them oops

**Comment by HeartSquared** [comment]
Looks like you need to update the test actually. It's apparently mocking the Icon from `component-library`

**Comment by HeartSquared** [comment]
Turns out there's another icon missing (the magnifying glass in the search) :')

image

File: .changeset/shiny-rivers-applaud.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": patch
+---
+
+Fix missing check icon for FilterMultiSelect options
```
Comment (HeartSquared) [inline .changeset/shiny-rivers-applaud.md:5]: Might need some extra info 😉

File: packages/components/src/SVG/icons/CheckIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { SVG, SVGProps } from "~components/SVG"
+
+export const CheckIcon = (props: Omit<SVGProps, "children">): JSX.Element => (
```
Comment (HeartSquared) [inline packages/components/src/SVG/icons/CheckIcon.tsx:4]: `displayName` 😗

File: packages/components/src/SVG/icons/SearchIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { SVG, SVGProps } from "~components/SVG"
+
+export const SearchIcon = (props: Omit<SVGProps, "children">): JSX.Element => (
```
Comment (HeartSquared) [inline packages/components/src/SVG/icons/SearchIcon.tsx:4]: `displayName` 😗

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**